src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -350,9 +352,11 @@ private static string CatchErrors(IReadOnlyCollection<Exception> errors)
 					fullText.WriteLine($"\tSource: {exception.TargetSite?.DeclaringType?.AssemblyQualifiedName}");
 					fullText.WriteLine($"\tMessage: {exception.Message}");
 					fullText.WriteLine($"\tTrace: {exception.StackTrace}");
-					fullText.WriteLine($"\tLocation: {frame.GetFileName()}");
-					fullText.WriteLine(
-						$"\tMethod: {frame.GetMethod()} ({frame.GetFileLineNumber()}, {frame.GetFileColumnNumber()})");
```

Comment:
I got a null pointer exception when frame is used. Should it be checked for null ?